docs/rackhd/ubuntu_package_installation.rst

```diff
@@ -0,0 +1,232 @@
+Ubuntu Package Based Installation
+---------------------------------
+
+
+Prerequisites
+~~~~~~~~~~~~~~~
+
+Start with an Ubuntu trusty instance with 2 nics:
+
+* eth0 for the 'public' network - providing access to RackHD APIs, and providing
+  routed (layer3) access to out of band network for machines under management
+
+* eth1 for dhcp/pxe to boot/configure the machines
+
+edit the network:
+
+* eth0 - assign IP address as appropriate for the environment, or you can use DHCP
+
+* eth1 static ( 172.31.128.0/22 )
+
+  this is the 'default'. it can be changed, but more than one file needs to be changed.)
+
+Install the prerequisite packages:
+
+.. code::
+
+    sudo apt-get install rabbitmq-server
+    sudo apt-get install mongodb
+    sudo apt-get install snmp
+    sudo apt-get install ipmitool
+    sudo apt-get install nodejs nodejs-legacy npm
+
+    sudo apt-get install ansible
+    sudo apt-get install apt-mirror
+    sudo apt-get install amtterm
+
+    sudo apt-get install isc-dhcp-server
+
+Set up the RackHD bintray repository for use within this instance of Ubuntu
+
+.. code::
+
+    echo "deb https://dl.bintray.com/rackhd/debian trusty main" | sudo tee -a /etc/apt/sources.list
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
+    sudo apt-get update
+
+Install the RackHD Packages. Note: these packages are rebuilt on every commit to master and are
+not explicitly versioned, but intended as a means to install or update to the latest code most
+conveniently.
+
```

Comment:
The debians deployed to bintray are just updates of master commits;  that was done intentionally to avoid potential bintray limits, but also make them available externally.  The HWIMO-BUILD does timestamp it's builds,  but are only deployed to an internal artifact repository.